examples/options/include/traccc/options/truth_finding.hpp

```diff
@@ -0,0 +1,23 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2023-2024 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+#include "traccc/definitions/common.hpp"
+#include "traccc/options/details/interface.hpp"
+
+namespace traccc::opts {
+class truth_finding : public interface {
+
+    public:
+    float m_min_pt;
```

Comment:
I'm honestly not sure, but wouldn't it be better to specify the default value here? We definitely do it like that in a number of places.
  - https://github.com/acts-project/traccc/blob/main/examples/options/include/traccc/options/accelerator.hpp#L23
  - https://github.com/acts-project/traccc/blob/main/examples/options/include/traccc/options/detector.hpp
  - etc.

Reply:
What happened here? I would still prefer to keep the defaults in the header, and not hidden in the implementation.